jhu-idc/idc-isle-dc
# Remove the "name" module configuraton

* Removes the configuration for the `name` module, which was removed in an earlier commit
* Updates to the latest buildkit images to include a Drupal image that will terminate the container upon startup error
* Updates the snapshot to contain the updated config


# To test
* Verify github actions passes.  There's no point in running this PR locally
* Go to the tip of `development`, and specify in `.env` the value of `TAG` from this PR:  `upstream-20200824-f8d1e8e-14-g09641e3`.
* do `make static-docker-compose.yml` and `make up`.  You should encounter an error.  The reason:  the `drupal` image in `upstream-20200824-f8d1e8e-14-g09641e3` will now stop the container with an error code if the Drupal bootup sequence fails.  Drupal is happy in this PR because the configuration is fixed.  The tip of `development` does not have the fix, therefore ought to fail with a Drupal image that performs a check.

Tests for this functionality will be in a subsequent PR 

resolves #22
